app/views/layouts/application_v2.html.erb

```diff
@@ -3,6 +3,7 @@
   <head>
     <title>Amplify New Styling Demo</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/all.css" integrity="sha384-lKuwvrZot6UHsBSfcMvOkWwlCMgc0TaWr+30HWe3a4ltaBwTZhyTEggF5tJv8tbt" crossorigin="anonymous">
```

Comment:
Can we host our assets ourselves instead of relying on the CDN?

Reply:
Eventually yes, this is a fairly temp demo for the UI so setting this up quickly for now for approval